mmil.raw.html

```diff
@@ -11455,6 +11461,49 @@
   <td>the set.mm proof uses sineq0</td>
 </tr>
 
+<tr>
+  <td>cosord</td>
+  <td>~ cosordlem , ~ cos11</td>
+  <td>presumably provable but the set.mm proof
+  uses real number trichotomy</td>
+</tr>
+
+<tr>
+  <td>sinord</td>
+  <td><i>none</i></td>
+  <td>the set.mm proof depends on cosord</td>
+</tr>
+
+<tr>
+  <td>recosf1o</td>
+  <td>~ ioocosf1o</td>
+  <td>the set.mm proof of recosf1o relies on ivthle2</td>
+</tr>
+
+<tr>
+  <td>resinf1o</td>
+  <td><i>none</i></td>
+  <td>the set.mm proof relies on recosf1o</td>
+</tr>
+
+<tr>
+  <td>tanord1</td>
+  <td><i>none</i></td>
+  <td>the set.mm proof relies on ltord1</td>
+</tr>
+
+<tr>
+  <td>tanord</td>
+  <td><i>none</i></td>
+  <td>the set.mm proof relies on tanord , real number trichotomy, and ltord1</td>
```

Comment:
tanord relies on tanord?

Reply:
Good eyes. The second one should be `tanord1`. I've updated the pull request with that change.